tests/ply_wise_data_test.py

```diff
@@ -0,0 +1,140 @@
+from collections.abc import Sequence
+
+from ansys.dpf.gate.common import locations
+import numpy as np
+
+from ansys.dpf.composites.composite_model import CompositeModel
+from ansys.dpf.composites.constants import Sym3x3TensorComponent
+from ansys.dpf.composites.ply_wise_data import ReductionStrategy, get_ply_wise_data
+from tests.helper import get_basic_shell_files
+
+
+def get_reduced_value(all_spot_values: Sequence[float], reduction_strategy: ReductionStrategy):
+    if reduction_strategy == ReductionStrategy.AVG:
+        return sum(all_spot_values) / len(all_spot_values)
+    if reduction_strategy == ReductionStrategy.MIN:
+        return min(all_spot_values)
+    if reduction_strategy == ReductionStrategy.MAX:
+        return max(all_spot_values)
+    if reduction_strategy == ReductionStrategy.BOT:
+        return all_spot_values[0]
+    if reduction_strategy == ReductionStrategy.MID:
+        return all_spot_values[2]
+    if reduction_strategy == ReductionStrategy.TOP:
+        return all_spot_values[1]
+    raise ValueError(f"Unknown reduction strategy: {reduction_strategy}")
+
+
+ALL_REDUCTION_STRATEGIES = [
+    ReductionStrategy.AVG,
+    ReductionStrategy.MIN,
+    ReductionStrategy.MAX,
+    ReductionStrategy.BOT,
+    ReductionStrategy.MID,
+    ReductionStrategy.TOP,
+]
+
+ALL_TENSOR_COMPONENTS = [
+    Sym3x3TensorComponent.TENSOR11,
+    Sym3x3TensorComponent.TENSOR22,
+    Sym3x3TensorComponent.TENSOR22,
+    Sym3x3TensorComponent.TENSOR21,
+    Sym3x3TensorComponent.TENSOR31,
+    Sym3x3TensorComponent.TENSOR32,
+]
+
+
+def get_all_spot_values_first_element_first_node(stress_field, component=0):
```

Comment:
I see that a lot of these methods don't have docstrings yet.

Reply:
Hi @PipKat Thanks for the review. These function are just helper functions for tests and are not part of the public API. Therefore they don't need documentation.